src/FoundryCSS/ui/blocks/Hero/_hero.scss

```diff
@@ -2,15 +2,17 @@
  * Hero:
  *
  * Large page headers containing a background color or background image.
+ *
+ * !! This module @extends %background--cover from FoundryCSS/ui/utilities/backgrounds.scss !!
  */
 
 @import 'hero-config';
 
 %hero,
 .hero {
     position: relative;
-    padding: map-get($hero-sizes, 'normal');
-    @extend %background--cover;
+    padding: map-get($hero-sizes, 'default');
+    @extend %background--cover !optional;
 
```

Comment:
Extend first, then overriding rules.
Also, did we ever resolve the extend - source order issue?

Reply:
We didn't resolve it per se as we couldn't recreate, but we suspect it was caused by extending the .class rather than %class. One to keep an eye on and flag up if it happens again.